lib/docker-utils.coffee

```diff
@@ -335,4 +335,9 @@ class RdtDockerUtils
 					else
 						throw new Error("Unsupported driver: #{dockerInfo.Driver}/")
 
+	isBalena: =>
+		@docker.versionAsync().get('Engine')
+		.then (engine) ->
+			return engine == 'balena'
```

Comment:
In https://github.com/resin-io/docker-progress/blob/master/index.coffee#L34 there is a check for `balaena` as well - do we have any released resinOS versions with `balaena` as the engine version?

Reply:
Not that I know of - balena was introduced into resinOS only a few days ago https://github.com/resin-os/meta-resin/pull/902 so even if a previous version identified itself as "balaena", only the one that identifies itself as "balena" would have a balena.pid, anything else would still use docker.pid